src/style.css

```diff
@@ -0,0 +1,120 @@
+@import url(./styles/stars.css);
+
+:root {
+  --color-main-text: #060b35;
+  --color-text: #616070;
+  --color-blue: #00acdc;
+  --color-white: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  line-height: 18px;
+  color: var(--color-main-text);
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background-color: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__icon {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__rating {
+  margin-top: 0;
+  margin-bottom: 24px;
+  line-height: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__reviews {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-main-text);
+}
+
+.card__title {
+  margin-top: 0;
+  margin-bottom: 4px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main-text);
+}
+
+.card__product-code {
+  margin-top: 0;
+  margin-bottom: 16px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text);
+}
+
+.card__price-block {
+  margin-top: 0;
+  margin-bottom: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-text {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--color-text);
+}
+
+.card__price-cost {
+  margin: 0;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--color-main-text);
+}
+
+.card__button-buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: var(--color-white);
+  text-decoration: none;
+  width: 166px;
+  height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Use paddings instead

Reply:
isn't fixed